### ISSUE TITLE
Alt text for media content images

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/sul-embed#1755]
Currently, neither the Thumbnail nor Full (non-thumbnail) `<img>` elements have `alt` attributes. After consulting with the POs (@dinahhandel  @andrewjbtw ) the near-term course of action for image alt text is as follows.

- **Thumbnail image display**: The thumbnail-display `<img>` elements in the Media Contents panel should have a null alt: `alt=""` or simply `alt`. (This is acceptable because the relevant information is already conveyed by the link associated with each image. Repeating this information as alt text would be redundant.)
- **Full image display**: The full-display `<img>` elements in the viewer Body should have alt text set to the image's title in the metadata. That is, the same title as seen in the Media Contents panel, such as `"Image of media (1 of 2)"` in [this example](https://sul-purl-stage.stanford.edu/xb302rj8598).
